rollup.config.mjs

```diff
@@ -0,0 +1,28 @@
+import commonjs from '@rollup/plugin-commonjs';
+import terser from '@rollup/plugin-terser';
+
+export default [{
+  input: './index.mjs',
+  output: {
+    file: 'dist/eventemitter3.esm.js',
```

Comment:
Can we also create a minified version ESM version?

Reply:
Sure. btw, how do you want to approach it?

If we just bundle the file, it is effectively the same as `dist/index.mjs`, so if you want, we can just use that as the default export